server/config.ts

```diff
@@ -6,7 +6,7 @@ function get<T>(name: string, fallback: T, options = { requireInProduction: fals
   if (process.env[name]) {
     return process.env[name]
   }
-  if (fallback !== undefined && (!production || !options.requireInProduction)) {
+  if (fallback !== undefined && !(production && options.requireInProduction)) {
```

Comment:
DeMorgan's law makes it easier to understand, at least for me

Reply:
oh my. Don't like either much but yours is definitely better.